install_pim/manual/crontab_tasks_ee.rst.inc

```diff
@@ -38,6 +39,7 @@ In the following example, these lines will run the versioning calculation at 5am
 
     # m  h  dom  mon  dow  command
     0 5   *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console pim:versioning:refresh --env=prod > /path/to/installation/pim-enterprise-standard/var/logs/refresh_versioning.log 2>&1
+    1 *   *    *    *    php /path/to/installation/pim-enterprise-standard/bin/console bin/console akeneo:connectivity-audit:update-data --env=prod > /path/to/installation/pim-enterprise-standard/var/logs/refresh_audit_data.log 2>&1
```

Comment:
The sentence 

`In the following example, these lines will run the versioning calculation at 5am, the rules calculation at 6pm and the aggregate volume at 10pm every day:`

is not up to date. It misses this job. Maybe it's not mandatory to add this line for every jobs btw (it's not done for quality insight jobs).